.github/workflows/upload_to_pypi.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4.2.0
       with:
-        python-version: '3.7'
+        python-version: '3.10'
```

Comment:
not 3.8 ?

Reply:
No, I tried to stream line this such that the latest is the default and we focus the testing of 3.8 on the old dependencies (then this does not have to change again when we drop 3.8)

Reply:
ah, cool!